### PR TITLE
Update enable-previews-of-on-premises-search-results-in-cloud-hybrid-…

### PR DESCRIPTION
**Pull request by tseward**
…search.md

Update for #858.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointServer/hybrid/enable-previews-of-on-premises-search-results-in-cloud-hybrid-search.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ Follow these procedures:
     
 2. [Configure Office Online Server for SharePoint Server](/officeonlineserver/configure-office-online-server-for-sharepoint-server-2016/configure-office-online-server-for-sharepoint-server-2016)
     
-3. Optional: Make the Office Online Server accessible outside the corporate network.
+3. Optional: Make the Office Online Server accessible outside the corporate network. This can be accomplished by configuring your corporate firewall to allow access from the public Internet to your Office Online Server. You must configure public DNS to match the `ExternalUrl` value in your Office Online Server farm. This value can be set when creating a farm using `New-OfficeWebAppsFarm -ExternalUrl <value>` or `Set-OfficeWebAppsFarm -ExternalUrl <value>`. You must use SSL when allowing access to Office Online Server from the Internet.
```
Comment (nenonix) [inline SharePoint/SharePointServer/hybrid/enable-previews-of-on-premises-search-results-in-cloud-hybrid-search.md]:
```suggestion
3. Optional: Make the Office Online Server accessible outside the corporate network. To configure your corporate firewall to allow access from the public Internet to your Office Online Server, set the public DNS to match the `ExternalUrl` value in your Office Online Server farm. This value can be set when creating a farm using `New-OfficeWebAppsFarm -ExternalUrl <value>` or `Set-OfficeWebAppsFarm -ExternalUrl <value>`. You must use SSL when allowing access to Office Online Server from the Internet.
```

Comment (nenonix) [inline SharePoint/SharePointServer/hybrid/enable-previews-of-on-premises-search-results-in-cloud-hybrid-search.md]:
```suggestion
3. Optional: Make the Office Online Server accessible outside the corporate network. You can configure your corporate firewall to allow access from the public Internet to your Office Online Server. To do this, set the public DNS to match the `ExternalUrl` value in your Office Online Server farm. This value can be set when creating a farm. using `New-OfficeWebAppsFarm -ExternalUrl <value>` or `Set-OfficeWebAppsFarm -ExternalUrl <value>`. You must use SSL when you give the Internet access to your Office Online Server.
```

Comment (tseward) [reply SharePoint/SharePointServer/hybrid/enable-previews-of-on-premises-search-results-in-cloud-hybrid-search.md]: You have a period after 'creating a farm' when there shouldn't be one based on the edit. The last sentence should not use the word 'give'. We're not giving the OOS farm anything :-)

Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/hybrid/enable-previews-of-on-premises-search-results-in-cloud-hybrid-search.md]:
```suggestion
3. Optional: Make the Office Online Server accessible outside the corporate network. You can configure your corporate firewall to allow access from the public Internet to your Office Online Server. To do this, set the public DNS to match the `ExternalUrl` value in your Office Online Server farm. This value can be set when creating a farm using `New-OfficeWebAppsFarm -ExternalUrl <value>` or `Set-OfficeWebAppsFarm -ExternalUrl <value>`. You must use SSL when allowing Internet access to your Office Online Server.
```

**Comment by scanum** [comment]
@tklarsen Hi Tone, changes look correct. Could you please check and merge? Thanks!

**Pull request merged**